src/migrations/1636110445512-clean-nameids.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+const tableNames = [
+  'user',
+  'project',
+  'organization',
+  'ecoverse',
+  'challenge',
+  'opportunity',
+];
+
+const buildQuery = (tableName: string) =>
+  `update ${tableName} set nameID = LOWER(REPLACE(nameID, '_', '-')) where nameID like '%_%'`;
```

Comment:
There are multiple other characters that may need to be replaced: (, ), ...suggest to get a list of all nameIDs from prod to see what characters are in nameIDs that should not be and then update this migration accordingly

Reply:
I checked production, there are messed up data with the following format:
- with _
- with ( and )
- without any firstname and lastname (captured in a separate story)
Replace _ with - and remove ( and ) and we should be good.